src/test/testdata/expected/lsl/literalArrays.lsl

```diff
@@ -0,0 +1,20 @@
+libsl "1.0.0";
+library literalArrays;
+typealias Int = int32;
+typealias ArrayType = array<Int>;
+automaton A : Int {
+    var arrayVariable: ArrayType;
+    fun f(param: Int) {
+        action TEST_ACTION([123]);
+        action TEST_ACTION([123]);
+        action TEST_ACTION([123, 321]);
+        action TEST_ACTION(["test string", param]);
+        action TEST_ACTION(["test string", param]);
+        action TEST_ACTION([]);
+        action TEST_ACTION([((1 + 2) + 3)]);
+    }
+    fun g() {
+        arrayVariable = [1, "2", "null"];
```

Comment:
I don't think it is a correct behavior. The `arrayVariable` is an array of integers as I can infer from aliases, not things of any type. The end-user should expect an error occurring during type checking.

Reply:
But we still have no type checking :)

So, yeah, when these diagnostics will be added, this code turns to invalid. I can fix the test If you wish